other/check_strong_password.py

```diff
@@ -0,0 +1,47 @@
+# This Will Check Whether A Given Password Is Strong Or Not
+# It Follows The Rule that Length Of Password Should Be At Least 8 Characters
+# And At Least 1 Lower, 1 Upper, 1 Number And 1 Special Character
+
+
+import re
+
+
+def strong_password_detector(password: str) -> str:
+    """
+    >>> strong_password_detector('Hwea7$2!')
+    'This is a strong Password'
+
+    >>> strong_password_detector('Sh0r1')
+    'Your Password must be at least 8 characters long'
+
+    >>> strong_password_detector('Hello123')
+    'Password should contain UPPERCASE, lowercase, numbers, special characters'
+    """
+
+    upper = re.compile(r"[A-Z]")
+    lower = re.compile(r"[a-z]")
+    num = re.compile(r"[0-9]")
+    spec_char = re.compile(r"[!@#$\^&\*\(\):;\'\"<>,\.\?\/|]")
+
+    if re.compile(r"\s").search(password) or len(password) < 8:
+        return "Your Password must be at least 8 characters long"
```

Comment:
Move these lines before line 22 so that we do not do the work on lines 22-24 if the password is too short.